### PR TITLE
Add storage option and use N5ZarrWriter.

### PR DESCRIPTION
**Pull request by kgabor**
Added ``--storage`` option that defaults to ``n5`` but allows now ``zarr`` to use ``N5ZarrWriter``.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -175,7 +175,6 @@
                     <groupId>net.java.dev.jets3t</groupId>
                 </exclusion>
             </exclusions>
-            <scope>provided</scope>
```
Comment (tpietzsch) [inline pom.xml:178]: I would make this line separate commit.

...and then let's discuss whether it should be included.  I guess the idea is that spark is not included in the fatjar. I'm not sure whether this change breaks that?

Comment (tpietzsch) [reply pom.xml:178]: It would be cool if we can achieve both, being able to run locally from IDE, and building fat-jar without spark. Maybe removing the `<scope>` achieves exactly that, but I'm not sure.

File: src/main/java/net/preibisch/bigstitcher/spark/AffineFusion.java [hunk 2]
```diff
@@ -40,20 +22,38 @@
...
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.concurrent.Callable;
+
 @SuppressWarnings("FieldMayBeFinal")
 public class AffineFusion implements Callable<Void>, Serializable
 {
 	private static final long serialVersionUID = 2279327568867124470L;
+	enum StorageType { n5, zarr }
```
Comment (tpietzsch) [inline src/main/java/net/preibisch/bigstitcher/spark/AffineFusion.java:46]: Add newline before the enum

**Comment by tpietzsch** [comment]
Looks good!

As noted in inline comment, I would pull out the `<scope>` change to a separate commit (and separate PR maybe?)

**Comment by kgabor** [comment]
Removed ``pom.xml`` modification from this PR.

**Review by tpietzsch — APPROVED** [review]
